packages/holocron/__tests__/ducks/load.spec.js

```diff
@@ -207,10 +207,11 @@ describe('loadModule', () => {
       },
     });
     const rebuildReducer = jest.fn();
-    const modules = iMap({ 'my-module': 'Preloaded MockModule' });
-    await expect(thunk(dispatch, getState, { rebuildReducer, modules })).resolves.toBe('Preloaded MockModule');
+    const Module = { holocron: { name: 'my-module', reducer } };
+    const modules = iMap({ 'my-module': Module });
+    await expect(thunk(dispatch, getState, { rebuildReducer, modules })).resolves.toBe(Module);
     expect(dispatch).toHaveBeenCalledWith({ type: MODULE_LOADED, moduleName: 'my-module' });
-    expect(rebuildReducer).toHaveBeenCalled();
+    expect(rebuildReducer).not.toHaveBeenCalled();
```

Comment:
should this still call `rebuildReducer` ?

Reply:
This is server side logic. Does this mean the reducer is built elsewhere to @JAdshead 's point?

Reply:
@JAdshead it shouldn't. It would only call `rebuildReducer` when a reducer is present in the config.

@mtomcal this would run for[ both browser and server on module load](https://github.com/americanexpress/holocron/blob/main/packages/holocron/src/ducks/load.js#L142-L147).